### ISSUE TITLE
Can not highligh pathways between nodes

### ISSUE THREAD

**Issue by schmidt-uni-leipzig** [k4l4m/BIS-D3-Graph#17]
How can I show all paths between two nodes? 

Trying this feature, I got the error 

> TypeError: d3.event.sourceEvent is null
> if (d3.event.sourceEvent.button !== 0) { // no left click

**Comment by k4l4m** [comment]
Browser? Does it work [here](http://k4l4m.github.io/BIS-D3-Graph/examples/big-graph.html)?

**Comment by schmidt-uni-leipzig** [comment]
My approach: I select a node in the graph by double-click. Then I selected another node by double-click. The following error is show in Firefox:

> TypeError: d3.event.sourceEvent is null
> if (d3.event.sourceEvent.button !== 0) { // no left click

I tested with the link above.

**Comment by k4l4m** [comment]
Yeah I am on the double click bug. To select a node just single click (not double click). I updated the wrong description of the graph command ([Commit](https://github.com/k4l4m/BIS-D3-Graph/commit/e9b98d026fe23554cd04faf272f8f71d691f685f)).
